server/metrics/metrics.go

```diff
@@ -158,91 +175,74 @@ func NewMetrics(info InstanceInfo) Metrics {
 	})
 	m.registry.MustRegister(m.httpErrorsTotal)
 
-	m.changeEventQueueRejectedTotal = prometheus.NewCounter(prometheus.CounterOpts{
+	m.changeEventsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   MetricsNamespace,
 		Subsystem:   MetricsSubsystemEvents,
-		Name:        "change_event_queue_rejected_total",
-		Help:        "The total number of change events rejected due to the activity queue size being full.",
-		ConstLabels: additionalLabels,
-	})
-	m.registry.MustRegister(m.changeEventQueueRejectedTotal)
-
-	m.changeEventTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
-		Namespace:   MetricsNamespace,
-		Subsystem:   MetricsSubsystemEvents,
-		Name:        "change_event_total",
-		Help:        "The total number of MS Teams change events received.",
-		ConstLabels: additionalLabels,
-	}, []string{"change_type"})
```

Comment:
No longer distinguishing "processed change events" from "change events", as the coverage was redundant.